tests/Browser/UserTest.php

```diff
@@ -0,0 +1,204 @@
+                ->type('password', 'lifull@123')
+                ->press('Login')
+                ->assertPathIs($data['path']['admin']);
+        });
+    }
+
+    /**
+     * Admin redirect to page create user
+     * case 1-2
+     *
+     * @return void
+     */
+    public function testClickMenuToPageCreate()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@createUser')
+                ->assertPathIs($data['path']['createUser']);
+        });
+    }
+
+    /**
+     * Admin redirect to page edit user
+     * case 1-3
+     *
+     * @return void
+     */
+    public function testClickMenuToPageEdit()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@editUser2')
+                ->assertPathIs('/admin/user/edit/2');
+        });
+    }
+
+    /**
+     * Admin show popup delete user
+     * case 1-4
+     *
+     * @return void
+     */
+    public function testPopupDeleteUser()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@deleteModal2')
+                ->waitFor('#myModal2')
+                ->assertSee('Are you sure you want to delete');
+        });
+    }
+
+    /**
+     * Checkbox no check and not submit
+     * case 2-12
+     *
+     * @return void
+     */
+    public function testNotSubmitWithCheckboxAllNoCheck()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@editUser2')
+                ->check('can_see', null)
+                ->check('can_delete', null)
+                ->assertPathIs('/admin/user/edit/2');
+        });
+    }
+
+    /**
+     * Checkbox checked but not submit
+     * case 2-13
+     *
+     * @return void
+     */
+    public function testNotSubmitWithCheckboxAll()
+    {
+        $admin = $this->accountAdmin();
+        $data = $this->getDataTest();
+
+        $this->browse(function (Browser $browser) use ($admin, $data) {
+            $browser->loginAs($admin)
+                ->visit($data['path']['listUser'])
+                ->click('@editUser2')
+                ->check('can_see')
+                ->check('can_delete')
+                ->assertPathIs('/admin/user/edit/2');
```

Comment:
'Not edit user, all data fields unchanged' , this is expected in test case , this arrert not tell me that part